README.md

```diff
@@ -1,52 +1,69 @@
+:sparkles: This has now been upgraded to cordova android@12.0.0 and iOS@6.2.0. It has also been upgraded to [android API 33 and the latest iOS versions](https://github.com/e-mission/e-mission-docs/issues/934), [cordova-lib@10.0.0 and the most recent node and npm versions](<insert link>). It also now supports CI, so we should not have any build issues in the future. __This should be ready to build out of the box.__
 
-Additional Documentation
----
+## Additional Documentation
 Additional documentation has been moved to its own repository [e-mission-docs](https://github.com/e-mission/e-mission-docs). Specific e-mission-phone wikis can be found here:
 https://github.com/e-mission/e-mission-docs/tree/master/docs/e-mission-phone
 
 **Issues:** Since this repository is part of a larger project, all issues are tracked [in the central docs repository](https://github.com/e-mission/e-mission-docs/issues). If you have a question, [as suggested by the open source guide](https://opensource.guide/how-to-contribute/#communicating-effectively), please file an issue instead of sending an email. Since issues are public, other contributors can try to answer the question and benefit from the answer.
 
-Updating the UI only
+:sparkles: Check [Contributing](#6-contributing) if you're interested in contributing for this project :sparkles:
+
+## Contents
+#### 1. [Creating logos](#1-creating-logos) 
+#### 2. [Updating the UI only](#2-updating-the-ui-only) 
+#### 3. [Updating the e-mission-* plugins or adding new plugins](#3-updating-the-e-mission--plugins-or-adding-new-plugins) 
+#### 4. [End to End Testing](#4-end-to-end-testing)
+#### 5. [Beta-testing debugging](#5-beta-testing-debugging) 
+#### 6. [Contributing](#6-contributing)
+
 ---
+
+## 1. Creating logos
+
+If you are building your own version of the app, you must have your own logo to
+avoid app store conficts. Updating the logo is very simple using the [`ionic
+cordova resources`](https://ionicframework.com/docs/v3/cli/cordova/resources/)
+command.
+
+**Note**: You may have to install the [`cordova-res` package](https://github.com/ionic-team/cordova-res) for the command to work
+
+## 2. Updating the UI only
 [![osx-serve-install](https://github.com/e-mission/e-mission-phone/workflows/osx-serve-install/badge.svg)](https://github.com/e-mission/e-mission-phone/actions?query=workflow%3Aosx-serve-install)
 
 If you want to make only UI changes, (as opposed to modifying the existing plugins, adding new plugins, etc), you can use the **new and improved** (as of June 2018) [e-mission dev app](https://github.com/e-mission/e-mission-devapp/) and install the most recent version from [releases](https://github.com/e-mission/e-mission-devapp/releases). 
 
 ### Installing (one-time)
 
-Run the setup script
+1. Run the setup script
 
 ```
-$ bash setup/setup_serve.sh
+bash setup/setup_serve.sh
```

Comment:
But then when you copy the command to your clipboard (with the copy button Github adds), the `$` gets included, annoyingly.
I regularly plenty of READMEs without `$` prefixing every command

Reply:
This seems like it is all the fault of the Github "copy" button 😄 
I am open to revisiting this but would like to see examples of the READMEs first.

Reply:
Would these README examples work?
- https://github.com/airbnb/javascript
- https://github.com/NationalSecurityAgency/ghidra

I don't mind restoring it but like @JGreenlee said, it can be annoying to manually select the command everytime

Reply:
The first is irrelevant since it includes **code** samples, not command line samples.

The second is in fact relevant. If you can find a second example of CLI samples, I am happy to switch.

Reply:
1. https://github.com/necolas/react-native-web/blob/master/.github/CONTRIBUTING.md
1. https://github.com/facebook/create-react-app/
1. https://github.com/enketo/enketo-transformer
1. TADA's certs repo

A lot of React / React Native docs are kept on their own website, not READMEs, but when they do show CLI samples they generally don't have `$` (e.g. https://callstack.github.io/react-native-paper/docs/guides/getting-started/)

Reply:
ok, I am convinced. @niccolopaganini You can keep remove the leading `$` in this PR.